.gitignore

```diff
@@ -8,3 +8,6 @@ __pycache__
 
 # ignore autogenerated .pyc files
 *.pyc
+
+#passwords files
+secret.py
```

Comment:
A non-python file would be better since contents in this file will vary from user to user.
Consider using a `.env` file